scripts/monorepo/checkIfPackagesAffected.js

```diff
@@ -0,0 +1,28 @@
+const getAffectedPackages = require('./getAffectedPackages');
+
+const FLUENTUI_REACT = '@fluentui/react';
+const FLUENTUI_REACT_COMPONENTS = '@fluentui/react-components';
+const FLUENTUI_REACT_NORTHSTAR = '@fluentui/react-northstar';
+const packages = [FLUENTUI_REACT, FLUENTUI_REACT_COMPONENTS, FLUENTUI_REACT_NORTHSTAR];
+
+const isPackageAffected = packagesToCheck => {
```

Comment:
What do you think about an alternative approach to this CLI:

```
$ check-affected @fluentui/react-northstar
```

That way you can rely on the return of the command to set the env variable in the pipeline
(yml code is definitely wrong, just to illustrate the idea)
```yml
- script: |
          $affected=check-affected @fluentui/react-northstar
          if [something]; then
            echo ##vso[task.setvariable variable=PackageAffected;isOutput=true]true
          fi
        displayName: Check if northstar is affected
```

This way the only place the env variable exists and is used is in our pipelines. And for now we don't need to have separate env variables for v8 and northstar

Reply:
good idea. less magic with the console.logs

Reply:
can be done as a followup PR

Reply:
I do like the idea of keeping the env variable within the pipeline file and can definitely do it in a follow up PR. However, i think we’d still need to create different env output variables for v8 and northstar in this case    since the check for packages affected happens in one job and both northstar and v8 jobs depend on that one job, they essentially share the same env variables.

Reply:
IMO the change is not that different, most of the work for the CLI command already exists so it's quite easy to refactor in this PR... but I won't block the PR for this 😄

Reply:
@ling1726 made the changes here: [80af4a7](https://github.com/microsoft/fluentui/pull/21770/commits/80af4a7b1bb485927893c9ad5cb88fc833dbd209). Although figuring out the correct bash syntax certainly wasn't as easy as I'd hoped 😆